mirgecom/mechanisms/__init__.py

```diff
@@ -67,5 +67,9 @@ def get_mechanism_cti(mechanism_name: str) -> str:
     mech_data = import_mechdata()
     mech_file = mech_data / get_mechanism_file_name(mechanism_name)
     with mech_file.open() as fp:
-        mech_cti = fp.read()
+        read_data = fp.read()
+    try:
+        mech_cti = read_data.decode()
+    except (UnicodeDecodeError, AttributeError):
+        return read_data
```

Comment:
This looks super fishy to me. How come Cantera needs us to do unicode decoding on its files?

Reply:
Cantera doesn't (seem to) care whether we pass in bytes or a string. The test cares, because it calls a string-specific built-in.  The real weird thing is that on ubuntu VM in CI, without this in-place, this routine returns a byte stream. (!)

Reply:
Could you add comments to state these things?

Also, maybe
```
if isinstance(read_data, bytes):
```
to clarify the intention?

Reply:
(b97443d) I'm with you, this seems fishy still.